### ISSUE TITLE
v0.54.0 is not installing plugins

### ISSUE THREAD

**Issue by DmitriyLewen** [aquasecurity/trivy#7273]
### Discussed in https://github.com/aquasecurity/trivy/discussions/7272

<div type='discussions-op-text'>

<sup>Originally posted by **fatihtokus** July 31, 2024</sup>
### Description

Latest version of trivy v0.54.0 is not installation trivy plugins

### Desired Behavior

Install plugins without an issue

### Actual Behavior

$ curl -s -L -o - https://github.com/aquasecurity/trivy/releases/download/v0.54.0/trivy_0.54.0_Linux-64bit.tar.gz | tar -xz
$ ./trivy plugin install referrer
2024-07-31T10:14:01Z	INFO	[plugin] Updating the plugin index...	url="https://aquasecurity.github.io/trivy-plugin-index/v1/index.yaml"
2024-07-31T10:14:01Z	INFO	[plugin] Installing the plugin...	src="github.com/aquasecurity/trivy-plugin-referrer"

### Reproduction Steps

```bash
1. trivy plugin install referrer
   trivy plugin list
2.
3.
...
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
trivy plugin install referrer
trivy plugin list
```

### Operating System

linux

### Version

```bash
v0.54.0
```

### Checklist

- [ ] Run `trivy clean --all`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)</div>

**Comment by knqyf263** [comment]
@DmitriyLewen I opened a PR, but it's a bit weird. It shows an error in my environment.

```
2024-07-31T14:51:48+04:00       FATAL   Fatal error     plugin install error: failed to install the plugin: unable to download the execution file (https://github.com/aquasecurity/trivy-plugin-referrer/releases/download/v0.3.1/trivy_plugin_referrer_0.3.1_macOS-ARM64.tar.gz): failed to download https://github.com/aquasecurity/trivy-plugin-referrer/releases/download/v0.3.1/trivy_plugin_referrer_0.3.1_macOS-ARM64.tar.gz: Get "https://github.com/aquasecurity/trivy-plugin-referrer/releases/download/v0.3.1/trivy_plugin_referrer_0.3.1_macOS-ARM64.tar.gz": failed to round trip: failed to get the file content: GET https://api.github.com/repos/aquasecurity/trivy-plugin-referrer/contents/releases/download/v0.3.1: 404 Not Found []
```

https://github.com/aquasecurity/trivy/pull/7274

**Comment by DmitriyLewen** [comment]
hm...
Your changes work correctly for me:
```bash
➜  trivy git:(fix/github_releases) ✗ ./trivy -d plugin install referrer
2024-07-31T17:04:52+06:00       DEBUG   Cache dir       dir="/Users/dmitriy/Library/Caches/trivy"
2024-07-31T17:04:52+06:00       INFO    [plugin] Installing the plugin...       src="github.com/aquasecurity/trivy-plugin-referrer"
2024-07-31T17:04:53+06:00       DEBUG   [plugin] Loading the plugin metadata...
2024-07-31T17:04:53+06:00       DEBUG   Installing the plugin...        path="/Users/dmitriy/.trivy/plugins/referrer"
2024-07-31T17:04:53+06:00       DEBUG   Platform found  os="darwin" arch="arm64"
2024-07-31T17:04:53+06:00       DEBUG   Downloading the execution file...       uri="https://github.com/aquasecurity/trivy-plugin-referrer/releases/download/v0.3.1/trivy_plugin_referrer_0.3.1_macOS-ARM64.tar.gz"
2024-07-31T17:04:56+06:00       INFO    [plugin] Plugin successfully installed  name="referrer" version="0.3.1"
➜  trivy git:(fix/github_releases) ✗ ./trivy plugin list
Installed Plugins:
  Name:    referrer
  Version: 0.3.1


```

**Comment by knqyf263** [comment]
OK, so let's merge my PR and cut v0.54.1.

**Comment by knqyf263** [comment]
> @DmitriyLewen I opened [a PR](https://github.com/aquasecurity/trivy/pull/7274), but it's a bit weird. It shows an error in my environment.

I meant v0.54.0 returned an error and fixed the problem, but the behavior in my environment looks a bit different from the original bug report. Anyway, it's good to fix the bug.

**Comment by DmitriyLewen** [comment]
> behavior in my environment looks a bit different from the original bug report

I think author just didn't insert the error.
I see the same error as in your env.

**Comment by knqyf263** [comment]
Hopefully, it was fixed in [v0.54.1](https://github.com/aquasecurity/trivy/releases/tag/v0.54.1).
